langserver/initialize.go

```diff
@@ -1,13 +1,10 @@
 package langserver
 
 import (
-	"context"
-
 	lsp "github.com/sourcegraph/go-lsp"
-	"github.com/sourcegraph/jsonrpc2"
 )
 
-func initialize(ctx context.Context, conn *jsonrpc2.Conn, req *jsonrpc2.Request) (result interface{}, err error) {
```

Comment:
the unparam linter found that none of the parameters are being used in the function